src/classes/User.js

```diff
@@ -1,13 +1,14 @@
 import Ingredient from './Ingredient';
 
```

Comment:
Should the User class be a child of the RecipeRepo class? I think it has all of its properties and it has it's methods, right?